HeliumExample/src/main/java/com/chartboost/heliumsdk/sampleapp/SplashActivity.kt

```diff
@@ -0,0 +1,31 @@
+package com.chartboost.heliumsdk.sampleapp
+
+import android.content.Intent
+import android.os.Bundle
+import android.widget.Toast
+import androidx.appcompat.app.AppCompatActivity
+import com.chartboost.heliumsdk.HeliumSdk
+
+class SplashActivity : AppCompatActivity() {
```

Comment:
I would avoid splash activity entirely and focus on the one screen, one activity setup, would be easier and you could avoid problems with activity context.

Reply:
No problem for me, I did another example just for banners and it has one activity only. Let's see what @cb-jpadilla and others say.

Reply:
Yes. Let's go with the suggestion.

Reply:
done